manifests/m/momento/cli/0.26.3/momento.cli.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: momento.cli
+PackageVersion: 0.26.3
+Platform:
+- Windows.Desktop
+InstallerType: wix
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install

```

```suggestion
UpgradeBehavior: install
AppsAndFeaturesEntries:
  - DisplayVersion: 0.26.3.0
```
As the package is marketed with `0.26.3` version but the actual DisplayVersion written to Registry is `0.26.3.0`, we're going to use AppsAndFeaturesEntries to provided 1:1 match with Registry.